M2/Macaulay2/m2/multilin.m2

```diff
@@ -16,11 +16,13 @@ hasNoQuotients Ring := (R) -> true
 getMinorsStrategy := (R, opts) -> (
      bareiss := 0;  -- WARNING: these must match the engine!!
      cofactor := 1;
+     dynamic := 2;
      strat := if opts.?Strategy then opts.Strategy else null;
      if strat === global Bareiss then bareiss
      else if strat === global Cofactor then cofactor
+     else if strat === global Dynamic then dynamic
```

Comment:
I suggest that "Dynamic" be the default.  Except I also suggest that if "Limit" or "First" is given, then use Cofactor.  When should Bareiss be the default?  I see very few cases where it is better...  Even when it is a bit better, it seems that Dynamic is only a bit worse.